alacritty.yml

```
@@ -312,6 +319,13 @@
   #
   #indexed_colors: []
 
+  # Transparent backgrounds with transparency enabled
```

This phrasing with the double "transparent" sounds a bit odd.

```suggestion
  # Transparent cell backgrounds
```

I'd suggest keeping the headline short like this to just encourage people to read the expanded explanation below instead of confusing them unnecessarily.

---

This is still unchanged?

---

Yeah, I've discarded `alacritty.yml` changes at some point, since I was updating config to test things.

---

So fix it again? I'm puzzled why you're re-requesting a review without addressing the comments.

---

The push failed I guess, should be fine now.